mock_drivers/rosbag_mock_drivers/src/MockControllerDriver.cpp

```diff
@@ -30,7 +41,11 @@ uint8_t MockControllerDriver::getDriverStatus()
 
 void MockControllerDriver::vehicleCmdCallback(const autoware_msgs::VehicleCmd::ConstPtr& msg)
 {
-  robot_enabled_ = true;  // If a command was received set the robot enabled status to true
+  //robot_enabled_ = true;  // If a command was received set the robot enabled status to true
+  CommandData command_data;
+  command_data.linear_x = msg.ctrl_cmd.linear_velocity;
+  command_data.angular_z = msg.ctrl_cmd.steering_angle;
```

Comment:
msg is a pointer as defined in this functions declaration. So you need to access its elements with ```msg->```
for example
```msg->ctrl_cmd.steering_angle```